.github/workflows/build-citus-community-nightlies.yml

```diff
@@ -29,18 +30,26 @@ jobs:
           - ol/7
           - debian/stretch
           - debian/buster
+          - debian/bullseye
```

Comment:
nice to see that we immediately started publishing nightlies for this distro!

Reply:
Fixed

Reply:
I guess we will get rid of travis job after merging this pr, right ?

Reply:
I just meant that this is something very nice to see, so there wasn't anything to fix here :)

Reply:
Sorry I misread it